pages/common/coproc.md

```diff
@@ -0,0 +1,28 @@
+# coproc
+
+> BASH builtin for creating interactive asynchronous subshells.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#Coprocesses>.
+
+- Run a subshell asynchronously:
+
+`coproc { {{commands}}; }`
```

Comment:
```suggestion
- Run a subshell asynchronously:

`coproc { {{command1; command2; ...}}; }`
```

Mandatory fix: [ellipsis syntax must be used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).